container/health_check.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "hello-openshift4",
+    "kind": "Pod",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "hello-openshift",
+        "labels": {
+            "name": "hello-openshift"
+        }
+    },
+    "spec": {
+        "containers": [
+            {
+                "name": "hello-openshift",
+                "image": "zzhao/hello-openshift",
```

Comment:
this image.. can we have iamges hosted by some shared openshift-qe account and not individual people?